app/src/main/res/values/strings.xml

```diff
@@ -205,18 +205,18 @@
     <string name="toastmessage_timer_cancelled">Sleep timer cancelled.</string>
     <string name="toastmessage_timer_duration_increased">Duration increased.</string>
     <string name="toastmessage_sleep_timer_unable_to_start">Please start playback first.</string>
-    <string name="toastalert_unable_to_connect">Unable to connect. Device does not have an internet connection or stream may be outdated.</string>
+    <string name="toastalert_unable_to_connect">Unable to connect. Device does not have an internet connection or the stream is outdated.</string>
     <string name="toastalert_rename_unsuccessful">Unable to rename station. Please choose a different name.</string>
-    <string name="toastalert_delete_successful">The station was removed.</string>
+    <string name="toastalert_delete_successful">Station removed.</string>
     <string name="toastalert_no_external_storage">Unable to access external storage.</string>
-    <string name="toastalert_no_image_picker">No image picker available. Consider installing a Gallery application.</string>
+    <string name="toastalert_no_image_picker">No image picker available. Consider installing a Gallery app.</string>
     <string name="toastalert_headphones_unplugged">Headphones unplugged. Playback stopped.</string>
     <string name="toastalert_permission_denied">Permission denied:</string>
     <string name="toastalert_station_not_found">Unable to find station.</string>
-    <string name="snackbar_request_storage_access">Transistor needs to be granted read permissions to be able access data from external storage.</string>
+    <string name="snackbar_request_storage_access">Grant \"Read storage\" permission to access data from external storage.</string>
```

Comment:
Is this just the one permission?

Reply:
On the [code](https://github.com/y20k/transistor/blob/v4.0.10/app/src/main/java/org/y20k/transistor/PlayerFragment.kt#L455) level this permission is called "[READ_EXTERNAL_STORAGE](https://developer.android.com/reference/android/Manifest.permission#READ_EXTERNAL_STORAGE)".